compiler/lib/eval.ml

```diff
@@ -31,14 +31,14 @@ let get_static_env s = try Some (Hashtbl.find static_env s) with Not_found -> No
 
 module Int = Int32
```

Comment:
This could be moved inside eval_prim as a first class module based on backend using Int31 when backend is wasm. One would have to complete Int31. Having such a standalone implementation will help with testing.

Reply:
Done.